app/src/main/java/com/zulip/android/activities/DevAuthActivity.java

```diff
@@ -58,9 +61,19 @@ protected void onCreate(Bundle savedInstanceState) {
         authEmailAdapter.setOnItemClickListener(new AuthClickListener() {
             @Override
             public void onItemClick(String email) {
-                AsyncLogin asyncLogin = new AsyncLogin(DevAuthActivity.this, email, null, true);
-                asyncLogin.execute();
-                connectionProgressDialog.show();
+                if(email.contains("@")) {
+                    getApp().setEmail(email);
+                }
+                getServices()
+                        .loginDEV(email)
+                        .enqueue(new DefaultCallback<LoginResponse>() {
+                            @Override
+                            public void onSuccess(Call<LoginResponse> call, Response<LoginResponse> response) {
+                                super.onSuccess(call, response);
```

Comment:
if this method is a no-op it's better not to call it.  this prevents the possibility of the addition of logic to the super method affecting all sub-methods.